lib/error_messages.rb

```diff
@@ -0,0 +1,7 @@
+class ErrorMessages
```

Comment:
What if we have a warning message? Or log message? My suggestion as simple as `Message`

Reply:
my opinion is that these methods are useless and do not carry any benefit, which I noted a little above. The name leash reflects my attitude towards this useless thing)